### ISSUE TITLE
Schedule export

### ISSUE THREAD

**Issue by cubizh** [fw42/cubecomp#183]
It would be interesting to be able to export the schedule into cubecomps format (described at the bottom of http://cubecomps.com/make_your_public_page_looks_great.php)

**Comment by fw42** [comment]
Yeah good idea. If you want to learn some Ruby on Rails, this would be a good task to get started :-)

Otherwise I can do it too. Shouldn't take too long.
